lib/render/layout.php

```diff
@@ -499,8 +499,10 @@ function RenderThemeSelector()
 function RenderPaginator($numItems, $perPage, $offset, $urlPrefix)
 {
     if ($offset > 0) {
+        echo "<a title='First' href='${urlPrefix}0'>&#x226A;</a>&nbsp;";
```

Comment:
Alternatively: `&laquo;`

Reply:
`&laquo;` is `&#x00AB;`: «
`&#x226A;` is ≪ (much less than mathematic symbol)
The larger one looks better next to the normal less than symbol. Here's the laquo/raquo:
![image](https://user-images.githubusercontent.com/32680403/170501430-f3dd8309-edc1-4215-9ec9-b515c9d44e13.png)

Reply:
Makes sense :) Was pointing it out for completeness' sake as laquo and raquo are already used in other places.
There are `&lsaquo;` and `&lsaquo;` - the single quote variants, too, which would match the double variants.
Then again, they are quite small.